src/sweeper/__main__.py

```diff
@@ -75,13 +75,12 @@ def execute_sweepers(closet, try_fix):
         if tool.table_name:
             report = tool.sweep()
 
-            reports.append(report)
-
             if try_fix:
                 tool.try_fix()
                 #: run sweeper again to ensure all errors were fixed.
                 report = tool.sweep()
 
+            reports.append(report)
```

Comment:
Does this make sense? If you ran sweep with try fix, you would only get the report from the fix. Do we need the reports.append in both locations?

Reply:
I think that @gregbunce's code is working as is because both `sweep()` and `try_fix()` modify the instance property `report` and then `sweep()` returns that value. I wonder if both `sweep()` and `try_fix()` should return their own reports and not modify the same value. Something like:

```python
class Sweeper():
    def __init__(self, workspace, table_name, field_name):
        #: remove self.report property
        self.workspace = workspace
        self.table_name = table_name
        self.field_name = field_name

    def sweep(self):
        report = {'title': 'Address Test', 'feature_class': table_name, 'issues': []}

        return report
    
    def try_fix(self):
        report = {'title': 'Address Try Fix', 'feature_class': table_name, 'issues': []}

        return report
```